### PR TITLE
Add axe-matchers for accessibility testing

### PR DESCRIPTION
**Pull request by hartsick**
Runs accessibility checks on existing full-flow feature specs. Also includes fixes to make accessibility checks pass.

Can be skipped by setting SKIP_ACCESSIBILITY_SPECS="true".

This branch will fail until [this PR](https://github.com/codeforamerica/cfa-styleguide-gem/pull/19) gets merged into master for the styleguide and we update to latest version of the gem. [edit: updated! should be good]

[#160629497]

### CODE REVIEW

File: app/assets/stylesheets/overrides.scss [hunk 1]
```diff
@@ -55,15 +55,8 @@
...
-  margin-bottom: 0;
-}
-
-.range-text {
-  margin: 0 1em;
-}
-
-.form-divider {
-  margin: 1em;
+.main-footer {
```
Comment (hartsick) [inline app/assets/stylesheets/overrides.scss]: This is two changes in one: 1) moving non-override CSS to custom.scss, and 2) upping the contrast on our footer links to make a11y specs pass.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <%= render "analytics/ganalytics" %>
 
     <title>ColoradoBenefits</title>
```
Comment (luigi) [inline app/views/layouts/application.html.erb]: This isn't a11y-related, but we should change `<title>` to Report Changes Colorado.

Comment (hartsick) [reply app/views/layouts/application.html.erb]: good catch!

Comment (hartsick) [reply app/views/layouts/application.html.erb]: I made a really small change to this for now, and filed a ticket to improve across the site: https://www.pivotaltracker.com/story/show/161669616

**Pull request merged**